symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/util/PresentationMLParser.java

```diff
@@ -32,13 +32,13 @@
    * @param trim            flag if we want to trim the text result
    * @return the message text content extracted from the given PresentationML
    */
-  public static String getTextContent(String presentationML, Boolean trim) throws PresentationMLParserException {
+  public static String getTextContent(String presentationML, boolean trim) throws PresentationMLParserException {
```

Comment:
Are we sure we never put `null` as `trim` parameter ?

Reply:
Putting null there should not compile IMHO but let me put it back to Boolean. Basically I changed it to fix a sonar warning.

Reply:
This is breaking the API contract :)